app/src/androidTest/java/com/sensor_emu/DaoInstrumentedTest.java

```diff
@@ -25,13 +44,29 @@
 public class DaoInstrumentedTest {
 
     private EnergyDao energyDao;
+    private HumidityDao humidityDao;
+    private PressureDao pressureDao;
+    private TemperatureDao temperatureDao;
+    private EnergyFrequencyDao energyFrequencyDao;
+    private HumidityFrequencyDao humidityFrequencyDao;
+    private PressureFrequencyDao pressureFrequencyDao;
+    private TemperatureFrequencyDao temperatureFrequencyDao;
+    private TimestampDao timestampDao;
     private AppDatabase db;
 
     @Before
     public void createDb() {
         Context context = ApplicationProvider.getApplicationContext();
         db = Room.inMemoryDatabaseBuilder(context, AppDatabase.class).build();
         energyDao = db.getEnergyDao();
+        humidityDao = db.getHumidityDao();
+        pressureDao = db.getPressureDao();
+        temperatureDao = db.getTemperatureDao();
+        energyFrequencyDao = db.getEnergyFrequencyDao();
+        humidityFrequencyDao = db.getHumidityFrequencyDao();
+        pressureFrequencyDao = db.getPressureFrequencyDao();
+        temperatureFrequencyDao = db.getTemperatureFrequencyDao();
+        timestampDao = db.getTimestampDao();
```

Comment:
I was wondering if we could have only 1 method for accessing Dao implementations in `AppDatabase` using `Class` as parameter:

```java
fooClassDao = db.getDao(FooClass.class);
barClassDao = db.getDao(BarClass.class);
```

AppDatabase:
```java
public abstract GenericDao getDao(final Class class);
```